src/main/java/Main.kt

```diff
@@ -0,0 +1,13 @@
+import com.elbekD.bot.Bot
```

Comment:
I suggest, that we name folders as `src/main/kotlin`. 

That's the way it's handled in the following projects:

+ [ktlint](https://github.com/pinterest/ktlint/tree/master/ktlint/src/main)

+ [codecov example](https://github.com/codecov/example-kotlin)